README.rst

```diff
@@ -36,16 +36,14 @@ The required dependencies to use the software are:
 
 * Python >= 2.6,
 * setuptools
-* Numpy >= 1.6
+* Numpy >= 1.6.1
 * SciPy >= 0.9
 * Scikit-learn >= 0.12.1
-* Nibabel >= 1.1.0.
+* Nibabel >= 1.3.0
```

Comment:
1.3.0 is quite old according to piprot:

```
~/dev/nilearn $ piprot <(echo nibabel==1.3.0)
nibabel (1.3.0) is 819 days out of date. Latest is 2.0.0
Your requirements are 819 days out of date
```

Also this is the version that neurodebian on Ubuntu 12.04 comes with.